tests/contrib/redis/test_redis.py

```diff
@@ -260,8 +311,26 @@ def test_env_user_specified_redis_service(self):
         span = self.get_spans()[0]
         assert span.service == "mysvc", span.service
 
-    @TracerTestCase.run_in_subprocess(env_overrides=dict(DD_SERVICE="app-svc", DD_REDIS_SERVICE="env-redis"))
-    def test_service_precedence(self):
+    @TracerTestCase.run_in_subprocess(
+        env_overrides=dict(DD_SERVICE="app-svc", DD_REDIS_SERVICE="env-redis", DD_TRACE_SPAN_ATTRIBUTE_SCHEMA="v0")
```

Comment:
nit: `env-redis` is a pretty strange service name, since `env` is its own tag. We don't want anyone to think they should set a service name like that. Maybe just switch it to some like `redis-custom-serice` ? Not sure why that made it in before. Also this probably from fear instilled in me from old support days and service naming chaos.
```suggestion
        env_overrides=dict(DD_SERVICE="app-svc", DD_REDIS_SERVICE="redis-custom-service", DD_TRACE_SPAN_ATTRIBUTE_SCHEMA="v0")
```